src/python/py/models/README.md

```diff
@@ -216,6 +218,27 @@ python3 -m onnxruntime_genai.models.builder -i path_to_local_folder_on_disk -o p
 python3 builder.py -i path_to_local_folder_on_disk -o path_to_output_folder -p precision -e execution_provider -c cache_dir_to_store_temp_files --extra_options use_qdq=1
 ```
 
+
+### Quant Provider
+This option is to select the specific INT4 quantization algo to use. Option avalable are nvidia_awq(Nvidia modelopt awq implimentation) and default
+
+```
+# From wheel:
+python3 -m onnxruntime_genai.models.builder -i path_to_local_folder_on_disk -o path_to_output_folder -p int4 -e execution_provider -c cache_dir_to_store_temp_files --extra_options quant_provider=nvidia_awq use_qdq=True
+# From source:
+python3 builder.py -i path_to_local_folder_on_disk -o path_to_output_folder -p precision -e execution_provider -c cache_dir_to_store_temp_files --extra_options quant_provider=nvidia_awq use_qdq=True
```

Comment:
How is `use_qdq` used with `nvidia_awq`? I only see `use_qdq` used for the default `quant_provider` case.

Reply:
current implementation of the doesn't take this parameter as input but in internal implementation quant_format is of type qdq. More importantly use_qdq export the fp16 model with opt 21 which is required in this case. so i specified in the command line